extension/js/common/api/email-provider/gmail/oauth.ts

```diff
@@ -0,0 +1,35 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
```

Comment:
Let's put this file in `common/api/authentication/oauth.ts`

Let's put google-auth in `common/api/authentication/google-oauth.ts`